package.json

```diff
@@ -203,7 +203,7 @@
     "webpack-cli": "3.2.3"
   },
   "engines": {
-    "node": ">=8",
+    "node": ">=10",
```

Comment:
❤️

Reply:
There should be a way to get Renovate take care of this and [`.nvmrc`](https://github.com/Automattic/jetpack/blob/21f90723048ef8a074da91d33650c4a94644d266/.nvmrc) btw. cc @brbrr

Reply:
It seems like node updates are enabled by default in renovate. It uses `lts` policy by default 8 and 10 are considered an `lts` versions. We can update the policy to use the latest `lts` version.

Reply:
https://github.com/Automattic/jetpack/pull/11716